### ISSUE TITLE
During decode process, instead of AgentRemoteId, AgentCircuitId is updated.

### ISSUE THREAD

**Issue by varshneyabhi** [bluecatengineering/dhcproto#65]
Instead of AgentRemoteId, AgentCircuitId is updated, during decode process. The link to issue in code is:
https://github.com/bluecatengineering/dhcproto/blob/44fe6c9115bb39b17db74b66bb357839da8f4c5c/src/v4/relay.rs#L137

It should be
```rust
          RelayCode::AgentRemoteId => {
                let len = d.read_u8()? as usize;
                let data = d.read_slice(len)?.to_vec();
                AgentRemoteId(data)
          }
```

**Comment by varshneyabhi** [comment]
It should be 
`RelayCode::AgentRemoteId => {
                let len = d.read_u8()? as usize;
                let data = d.read_slice(len)?.to_vec();
                AgentRemoteId(data)
            }`

**Comment by varshneyabhi** [comment]
MR is raised here: https://github.com/bluecatengineering/dhcproto/pull/66

**Comment by leshow** [comment]
Thank you!

**Comment by leshow** [comment]
closed in #66
